### ISSUE TITLE
new scroll method? focus stealing

### ISSUE THREAD

**Issue by tablatronix** [gepd/Deviot#225]
windows

Did you say you were testing a new scroll method, I don't remember is this was the same as before.

But my focus gets stolen when things are running like compile etc. focus switches to console while I am typing and upload finished etc.

**Comment by tablatronix** [comment]
Serial monitor also seems to not go all the way to the bottom, it seems to be off by several lines maybe 5 or so ?

**Comment by tablatronix** [comment]
hmm auto clean not working again

**Comment by tablatronix** [comment]
hmm,  maybe I am not running dev again, not sure how that could happen

**Comment by gepd** [comment]
It was that, you wasn't running the dev?

**Comment by tablatronix** [comment]
I think so, i noticed both my machines say deviot 
1.2.1 or something but saw you commited a new version a while ago. Ill double check manually later

**Comment by gepd** [comment]
yep, you are not in the dev branch: https://github.com/gepd/Deviot/commit/a379d55e31fd36c6081be701d8433a99cc6785b5

**Comment by tablatronix** [comment]
Ill have to see why its not updating automatically

**Comment by tablatronix** [comment]
Its a bug in pc

https://github.com/renerocksai/sublime_zk/issues/50#issuecomment-377724551

**Comment by tablatronix** [comment]
I am still having issues with this, If I try typing when uploading I have issues, sometimes when I compile it tries to compile the serial window etc.
Focus seems to move around for auto scrolling...
